x/profiles/client/cli/cli_relationships.go

```diff
@@ -190,6 +201,7 @@ func GetCmdQueryUserBlocks() *cobra.Command {
 	}
 
 	flags.AddQueryFlagsToCmd(cmd)
+	flags.AddPaginationFlagsToCmd(cmd, types.QueryUserRelationships)
```

Comment:
The `types.QueryUserRelationships` is for creating the relationship. I think it should be `types.QueryUserBlocks`. If it is not present, we should add one in the `key.go`.